app/controllers/students/base_controller.rb

```diff
@@ -8,7 +8,7 @@ def must_be_student
   end
 
   def current_team
-    @current_team ||= current_user.teams.first
+    @current_team ||= current_user.teams.last
```

Comment:
Yes, this could work. Haven't tried it myself yet. Would be awesome if you could check it and reply back here with your findings.